clients/vault/Cargo.toml

```diff
@@ -48,9 +48,12 @@ sp-runtime = {git = "https://github.com/paritytech/substrate", branch = "polkado
 sp-std = {git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24"}
 
 # Substrate Stellar Dependencies
-stellar-relay = {git = "https://github.com/pendulum-chain/stellar-experiments.git", branch = "85-collect-and-store-scp-messages-in-vault-client" }
+stellar-relay = {path = "../../stellar-relay" }
```

Comment:
so this is going to be in another PR, where I add the `stellar-relay` (aka the stellar-experiment) to spacewalk.

Reply:
so this draft PR will fail.